components/ResultsPageComponents/SurveyOverview/SurveyOverview.jsx

```diff
@@ -74,18 +75,21 @@ export default function SurveyOverview({
         Overview
       </Text>
       <Text fontWeight={400} fontSize="20px">
-        The Work Commute survey recorded results from 12 to 27 July 2022, with
-        responses from 162 people.
+        The Work Commute survey was conducted on 13 to 29 July 2022 and recorded
```

Comment:
we should align the survey date with what is in the infographic.

![image](https://user-images.githubusercontent.com/4408259/185038410-9fa04697-b874-4690-8d38-fb2f7e58de66.png)

Can we double check the dates with Aainaa @RCode-Blue ?

Reply:
Design team has confirmed that the dates are from 13th to 29th.